Modules/QuestieQuestLog.lua

```diff
@@ -0,0 +1,239 @@
+---@class QuestieQuestLog
+local QuestieQuestLog = QuestieLoader:CreateModule("QuestieQuestLog")
+
+local elvEnabled = IsAddOnLoaded("ElvUI")
+
+function QuestieQuestLog:IsElvSkinningEnabled(E)
+    return E.private.skins.blizzard.enable == true and E.private.skins.blizzard.quest == true
+end
+
+function QuestieQuestLog:ElvSkinFrames()
+    if elvEnabled == false then
+        return
+    end
+
+    local E, L, V, P, G = unpack(ElvUI)
```

Comment:
Could you give these variables proper names and replace the unused ones with a `_`? I personally have no clue what the unpack returns in this case.

Reply:
To be fair I also don't know what unpacking ElvUI returns exactly. But I'm assuming the E stands for the main ElvUI object. So i renamed it to elv.